### DISCUSSION TITLE
Add in configurable option to disable user management from the UI

### DISCUSSION THREAD

**Discussion by CurlyDev1** [grafana/grafana#69883] [General]
Our organisation configures Grafana to use generic OAuth and uses the role_attribute_path to set what role  a user has by looking at the values received in the access token,

In the Grafana UI admins are allowed to modify a user's role thus making it possible to override the role a user is assigned when they sign in, this makes tracking what role a user has problematic since it can come from the access token or overridden in the UI.

Can an option be added to disable the ability to set a user's role from the UI ? this should also disable the relevant API endpoint

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82642. Feel free to continue the discussion around this there. Thank you!
